src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -4249,5 +4477,48 @@ private static void ValidateCosmosDbSetup(TestServer server)
 
             return false;
         }
+
+        private static string GetSimpleReadGraphQLQuery()
+        {
+            return @"{
+                    book_by_pk(id: 1) {
+                       id,
+                       title,
+                       publisher_id
+                    }
+                }";
+        }
+
+        private static string GetSimpleGraphQLCreateMutation()
+        {
+            return @"mutation createbook{
+                        createbook(item: { title: ""Book #1"", publisher_id: 1234 }) {
+                            title
+                            publisher_id
+                        }
+                    }";
+        }
+
```

Comment:
do we really need separate helpers for request content that could be present in the test itself and not have this request be coupled to multiple tests? e.g. if you have to change this request to accommodate one of the two tests where this request is used, you'd have to also update test code in at least one place.

Reply:
would also be helpful in test method description or code comment, which part of the queries/mutations are counted towards query depth? Just the selection set? the mutation input?